games-action/openclonk-7.0

```diff
@@ -0,0 +1,111 @@
+	openclonk$secondaryArchSuffix = $portVersion
+	app:OpenClonk$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libjpeg_turbo$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libxrandr$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLEW$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libault$secondaryArchSuffix
+	lib:libat_1.0k$secondaryArchSuffix
+	lib:libc$secondaryArchSuffix
+	lib:libcairo-gobject$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libgcc_s$secondaryArchSuffix
+	lib:libgdk_pixbuf_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgthread_2.0$secondaryArchSuffix
+	lib:libixml$secondaryArchSuffix
+	lib:libm$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libpango_1.0$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libpthread$secondaryArchSuffix
+	lib:libstdc++$secondaryArchSuffix
+	lib:libthreadutil$secondaryArchSuffix
+	lib:libtinyxml$secondaryArchSuffix
+	lib:libupnp$secondaryArchSuffix
+	lib:libvorbisfile$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES"
+	haiku_devel$secondaryArchSuffix
+	devel:g++$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libjpeg_turbo$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libxrandr$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libGLEW$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libault$secondaryArchSuffix
+	devel:libat_1.0k$secondaryArchSuffix
+	devel:libc$secondaryArchSuffix
+	devel:libcairo-gobject$secondaryArchSuffix
+	devel:libcairo$secondaryArchSuffix
+	devel:libgcc_s$secondaryArchSuffix
+	devel:libgdk_pixbuf-2.0$secondaryArchSuffix
+	devel:libgio_2.0$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libgthread_2.0$secondaryArchSuffix
+	devel:libixml$secondaryArchSuffix
+	devel:libm$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libpango_1.0$secondaryArchSuffix
+	devel:libpangocairo_1.0$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libpthread$secondaryArchSuffix
+	devel:libstdc++$secondaryArchSuffix
+	devel:libthreadutil$secondaryArchSuffix
+	devel:libtinyxml$secondaryArchSuffix
+	devel:libupnp$secondaryArchSuffix
+	devel:libvorbisfile$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+        "
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+        "
+BUILD()
+{
+	mkdir -p build
+	cd build
+	cmake .. -DCMAKE_INSTALL_PREFIX=$prefix -DDATAROOTDIR=$dataDir \
+		-DENABLE_DESKTOP_ENTRY=OFF -DMANDIR=$manDir -DDOCDIR=$docDir
```

Comment:
Not sure about the other options, but using -DCMAKE_INSTALL_PREFIX=$appsDir should be enough to get the build starting ...